extension/packagemanagerextension/nbextension/methods.js

```diff
@@ -0,0 +1,239 @@
+
```

Comment:
For now you can keep the code as is, but as I asked before, it would be great if you could replicate all the other swan extensions organisation/compilation/etc